dashboard/factories.py

```diff
@@ -32,22 +33,47 @@
 class CachedCertificateFactory(DjangoModelFactory):
     """Factory for Certificate"""
     user = SubFactory(UserFactory)
-    data = FuzzyAttribute(lambda: dict(
-        grade=randint(60, 100), **{'key{}'.format(i): FAKE.text() for i in range(3)}
-    ))
     course_run = SubFactory(CourseRunFactory)
+    data = LazyAttribute(lambda x: {
+        "certificate_type": "verified",
+        "grade": randint(60, 100)/100.0,
+        "course_key": x.course_run.edx_course_key,
```

Comment:
in `edx_api.certificates.models.Certificate`, this key is `course_id` not `course_key`